hooks/pre-commit

```diff
@@ -182,9 +182,24 @@ else
 fi
 echo "${reset}"
 
+echo "Running unit tests in changed directories... "
+for dir in $(git diff --cached --name-only --diff-filter ACM |grep -e ^pkg/ -e ^cmd/ |xargs -r dirname |sort |uniq); do
```

Comment:
This would be neat to put in script in hack/, but I don't think it's right for a pre-commit hook. Shippable will automatically run all unit tests for every PR.